regtests/run_spark_sql.sh

```diff
@@ -48,14 +48,15 @@ cd ${REGTEST_HOME}
 
 export SPARK_VERSION=spark-3.5.3
 export SPARK_DISTRIBUTION=${SPARK_VERSION}-bin-hadoop3
+export SPARK_LOCAL_HOSTNAME=localhost #avoid VPN messy up driver local IP address binding
 
 ./setup.sh
 
 if [ -z "${SPARK_HOME}"]; then
   export SPARK_HOME=$(realpath ~/${SPARK_DISTRIBUTION})
 fi
 
-SPARK_BEARER_TOKEN="${REGTEST_ROOT_BEARER_TOKEN:-principal:root;realm:default-realm}"
+SPARK_BEARER_TOKEN="${REGTEST_ROOT_BEARER_TOKEN:-principal:root;realm:polaris}"
```

Comment:
Looks like this change is optional. It passed without the change for me locally.